content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Linear Activation Function'
+Description: 'The linear activation function, also known as the identity function, is where the activation is proportional to the input.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'****
+---
+
+The **linear activation function**, also known as the identity function, is one of the most straight-forward activation function,where the output of linear activation function is exactly same as input.
+In general , linear activation function simply add weighted sum with bias and pass it as output.
```

Comment:
This statement contains some grammatical errors and is unclear. Can you reword it?